Object Store/func.py

```diff
@@ -1,67 +1,85 @@
 import io
-import oci
-import re
 import os
+import gzip
 import json
-import sys
-import requests
 import logging
-import time
-import gzip
 
-from fdk import response
+import oci
+import requests
```

Comment:
nit: Python imports should be grouped by standard lib, third party, then local ([source](https://peps.python.org/pep-0008/#imports))

Reply:
should be grouped like this now with oci + requests being the 3rd party imports